public/scripts/upload.js

```diff
@@ -249,23 +308,36 @@ function dismissUploading() {
   uploading?.classList.add("d-none");
 }
 
-function findFileListItem(fileIndex) {
+function findFileListItem(fileUlid) {
   const fileListNode = document.querySelector(".file-list");
   const itemNodes = fileListNode.querySelectorAll(".file-list-item");
   for (const itemNode of itemNodes) {
-    if (itemNode.dataset.fileIndex === String(fileIndex)) {
+    if (itemNode.dataset.fileUlid === String(fileUlid)) {
       return itemNode;
     }
   }
   return null;
 }
 
-function appendInitialFileListItem(fileIndex, filename) {
+function appendInitialFileListItem(fileUlid, filename) {
   const fileListNode = document.querySelector(".file-list");
 
   const itemNode = document.createElement("li");
+
+  const itemNodeRight = document.createElement("div");
+  const itemNodeLeft = document.createElement("div");
```

Comment:
discussion: instead of `left` vs `right`, maybe we can use more meaningful variable names such as `stateView` vs `filenameView`

Reply:
how about `previewStateView` and `fileInfoView`, because the left part only preview state, the image upload state is in the right part

Reply:
Sounds great 🥇